Makefile

```diff
@@ -102,25 +102,26 @@ local-clean:
 	rm -rf test/images tmp $(PACKAGE_NAME).spec $(PACKAGE_NAME)*.rpm $(PACKAGE_NAME)*.tar.gz test/end-to-end/wdio_report
 
 # build VMs
-$(VM_IMAGE): local-clean rpm bots
+$(VM_IMAGE): local-clean rpm bots test/common
 	# VM running cockpit, composer, and end to end test container
 	bots/image-customize -v \
-		-r "echo $(BROWSER) > /tmp/BROWSER" \
+		-r "echo $(TEST_BROWSER) > /tmp/BROWSER" \
```

Comment:
I think this is obsolete. Or do you need this for some reason still?

Reply:
ack. Thanks!